src/drive/targets/services/qualificationMigration.js

```diff
@@ -7,6 +7,9 @@ import {
   queryFilesFromDate,
   getMostRecentUpdatedDate
 } from 'drive/lib/migration/qualification'
+import fetch from 'node-fetch'
```

Comment:
Is it really needed? If yes, you should add a dep to node-fetch and not relying on the fact that the dep is here because another dep needs it